doc/package.nix

```
@@ -32,16 +32,24 @@ let
     sed -i -e '/\*Declared by:\*/,+1d' $out
   '';
 in
-pkgs.stdenv.mkDerivation {
-  name = "wire-docs";
-  buildInputs = with pkgs; [
-    mdbook
-    mdbook-alerts
+pkgs.stdenv.mkDerivation (finalAttrs: {
+  pname = "wire-docs";
+  version = "3";
```

```suggestion
  name = "wire-docs";
```
consider using this instead, as `name = pname + version`

---

this breaks the pnpm deps section